src/GraphQLParser.Tests/ParserTests.cs

```diff
@@ -1082,9 +1144,14 @@ enum Colors {
 ) on QUERY
 ".Parse(new ParserOptions { Ignore = options });
             var defs = document.Definitions;
-            defs.Count.ShouldBe(7);
+            defs.Count.ShouldBe(8);
             var parseComments = options == IgnoreOptions.None;
```

Comment:
```suggestion
            var parseComments = options == IgnoreOptions.None || options == IgnoreOptions.Locations;
```

Reply:
You may want to check to see a similar change is needed anywhere else in the tests.

Reply:
Or this

```suggestion
            var parseComments = !options.HasFlag(IgnoreOptions.Comments);
```